ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -103,30 +100,38 @@
         }
         else
         {
-            if (Model.DuplicateSubmission)
+            if (@Model.Event.AnswerSubmissionEnd < DateTime.UtcNow)
+            {
+                <div class="alert alert-danger" role="alert">
+                    <h3>The event is over. You may check answers, but they will not count toward your ranking.</h3>
+                </div>
+            }
+
+            @if (Model.DuplicateSubmission)
             {
             <div class="alert alert-danger" role="alert">
                 <h4>Oops</h4>
                 <span>You've already submitted @Model.SubmissionText!</span>
             </div>
             }
-
-            <form method="post">
-                <div asp-validation-summary="ModelOnly" class="text-danger"></div>
-                <div class="form-group">
-                    <label class="control-label">Answer</label>
-                    <input autofocus name="submissionText" class="form-control" data-val="true" data-val-required="Your answer cannot be empty" />
-                    <span style="font-size:12px">Submitted answers will automatically be capitalized and stripped of non-alphanumeric characters</span><br />
-                    @Html.ValidationSummary(true)
-                    <span class="text-danger"><span class="field-validation-valid" data-valmsg-for="submissionText" data-valmsg-replace="true"></span></span>
+            <div class="row">
```

Comment:
Is there a reason this row was added?

Reply:
It's just moving the row that was previously defined on line 60